assets/tests/models/ladderVehicle.test.ts

```diff
@@ -37,128 +37,89 @@ describe("putIntoLanes", () => {
       {
         vehicle,
         x: 0,
-        y: 10,
+        y: () => 10,
         vehicleDirection: VehicleDirection.Up,
       },
       {
         vehicle,
         x: 0,
-        y: 20,
+        y: () => 20,
         vehicleDirection: VehicleDirection.Up,
       },
       {
         vehicle,
         x: 0,
-        y: 20,
+        y: () => 20,
         vehicleDirection: VehicleDirection.Down,
       },
       {
         vehicle,
         x: 0,
-        y: 10,
+        y: () => 10,
```

Comment:
Should these functions be taking an argument?

Reply:
It doesn't really matter since we aren't using the arguments and JS doesn't care, but I can add them if that makes things more clear.